integration_tests/docs_generate/models/_models.yml

```diff
@@ -785,6 +785,65 @@ models:
           The date and time the source file was ingested into the data warehouse or
           landed in cloud storage
 
+
+  - name: provider_attribution
+    config:
+      database: tuva
+      schema: input_layer
+      tags: input_layer
+      materialized: table
+    description: >
+      The provider_attribution table assigns a provider to each patient-member_month
+      of eligibility. The user may assign both a payer attributed provider (as
+      typically assigned by the payer in VBC contracts) as well as custom
+      attributed provider (typically determined by the user).
+      Fields related to each attributed provider (practice, organization,
+      line of business)are also present.
+    columns:
+      - name: patient_id
```

Comment:
Not trying to open up an old can of worms because we've talked about this in other input layer tables, but especially if we are building this around VBC, should this be member_id? In my experience most folks think of them as members, not patients.

Reply:
maybe a good idea long term?  I think it should probably match the member/patient id mnemonic we use in eligibility, since they are treated as the same entity logically when joined downstream.  Should we consider changing the name in eligibility now too?  I'm not sure if we have enough time to think through the ramifications of that change if we want to release today, do we still want to do that?

Reply:
Yeah, this is going back to our long ongoing conversation. Since one of the main goals of adding attributed provider is to see spend sliced by provider (in the financial_pmpm mart), we decided to go with patient_id for now because that is what is used in the financial_pmpm mart.

Reply:
Yep, agreed - especially since that is what is used downstream. Just decided to risk calling it out here again. ;)